### ISSUE TITLE
Evaluate 1.5/1.6 Support via New Implementation

### ISSUE THREAD

**Issue by JakeWharton** [JakeWharton/ActionBarSherlock#352]
A lot of applications that use the library still have a very active user base running android 1.6 (and even 1.5). Due to our pluggable system of implementations it should be possible to provide a plugin to support these older devices, perhaps in a partially diminished capacity (e.g., no animations).

**Comment by JakeWharton** [comment]
Due to time, market share, and personal hatred for the classloader on these devices this will not be attempted.

**Comment by r3gis3r** [comment]
For the record : 
As I can't drop 1.6 users support on CSipSimple which is opensource/GPL (and so will try to not leave users behind), I have forked the project and started support of 1.6. I decided to switch to ABS for next version.

It's already working on 1.6 for my needs (it took me 1day to do), but drops some features that I don't use for now. 
It's features about accessibility that I plan to bring back soon.

I did something the very fast (and crappy) way : using reflection for super calls and using a wrapper class for all other needs.
So it's probably not clean enough to be merged but if anybody else is in the same case than me, feel free to reuse it :
https://github.com/r3gis3r/ActionBarSherlock

@JakeWharton : 
I think that it would be interesting and very useful to use lint TargetApi annotation in ABS (it will ensure that you don't use a method not supported)
Also, the csipsimple project link, if you want to count it in apps using your great lib : http://code.google.com/p/csipsimple/ -- even if the release with ABS inside is not yet done but under dev in nightly builds ;) .
